test-apps/display-test-app/README.md

```diff
@@ -39,7 +39,7 @@ npm run start:servers
 
 ## Using display-test-app
 
-Currently, display-test-app only supports opening snapshot iModels from the local disk. If you define the `SVT_STANDALONE_FILENAME` environment variable to contain the absolute path to an existing iModel file on your machine, then upon startup, a viewport displaying the contents of this iModel will be displayed. Otherwise, on startup the toolbar will have a button allowing you to select an iModel to open.
+Currently, display-test-app only supports opening snapshot iModels from the local disk. If you define the `SVT_STANDALONE_FILENAME` environment variable to contain the absolute path to an existing iModel file on your machine, then upon startup, a viewport displaying the contents of this iModel will be displayed. Otherwise, on startup the toolbar will have a button allowing you to select an iModel to open. If the file opened from disk is a briefcase that retains its link to iModelHub (as opposed to a snapshot) and is opened in read-write mode, it can push and pull changesets.

```

Comment:
> Currently, display-test-app only supports opening snapshot iModels from the local disk

This sentence doesn't make any sense and should be removed.

> that retains its link to iModelHub

I don't know what this means. Maybe just "is a briefcase from iModelHub"?